index.html

```diff
@@ -181,64 +188,68 @@ <h1>Multi-Post Stories</h1>
     </section>
 
     <section class="about-myself">
-      <h1 class="header">
-        About Myself
-      </h1>
+      <div class="about-myself-wrapper">
+        <h1 class="header">
+          About Myself
+        </h1>
 
-      <div class="header-text">
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      </div>
```

Comment:
- [ ] Also this section still has the design dissimilar to the request one make sure this issue is fixed too as suggested by the previous reviewer.

**YOur about section:**
![Screenshot from 2022-07-16 18-56-50](https://user-images.githubusercontent.com/80612925/179365021-df99cd44-65d0-488a-be75-a40dc394d77b.png)

**THe Figma design:**
![image](https://user-images.githubusercontent.com/80612925/179365027-f7d7da73-e91c-40ef-9adf-ce24677aa2f2.png)